apps/web/src/services/advanced/components/WorkspaceGroupAddUsersModal.vue

```diff
@@ -0,0 +1,163 @@
+<script setup lang="ts">
+import { reactive } from 'vue';
+
+import {
+    PButtonModal, PFieldGroup, PSelectDropdown, PIconButton,
+} from '@cloudforet/mirinae';
+
+import { i18n } from '@/translations';
+
+import { showSuccessMessage } from '@/lib/helper/notice-alert-helper';
+
+import { WORKSPACE_GROUP_MODAL_TYPE } from '@/services/advanced/constants/workspace-group-constant';
+import { useWorkspaceGroupPageStore } from '@/services/advanced/store/workspace-group-page-store';
+import { useRoleFormatter } from '@/services/iam/composables/refined-table-data';
+
+
+
+const workspaceGroupPageStore = useWorkspaceGroupPageStore();
+const workspaceGroupPageState = workspaceGroupPageStore.state;
+
+const state = reactive({
+    loading: false,
+    roles: [{
+        // TODO: temp data
+        user_id: 'Kara_Herzog@yahoo.com',
+        name: 'asdas',
+        state: 'ENABLED',
+        label: 'Workspace Member',
+        role_type: 'WORKSPACE_OWNER',
+    },
+    {
+        type: 'showMore',
+        label: 'show more',
+    }],
+    users: [{
+        // TODO: temp data
+        user_id: 'Bradford_McDermott@hotmail.com',
+    }],
+});
+
+const handleConfirm = () => {
+    showSuccessMessage(i18n.t('IAM.WORKSPACE_GROUP.MODAL.ALT_S_ADD_USRES'), '');
+    workspaceGroupPageStore.closeModal();
+};
+
+const handleCancel = () => {
+    workspaceGroupPageStore.closeModal();
+};
+
+const handleClose = () => {
+    workspaceGroupPageStore.closeModal();
+};
```

Comment:
suggestion: How about combining them into a single function if they perform the same role?